### ISSUE TITLE
centralization

### ISSUE THREAD

**Issue by torgo** [w3ctag/societal-impact-questionnaire#11]
This came up in the privacy task force during the discussion of https://github.com/w3ctag/privacy-principles/issues/261 - what can we say here about dangers of centralization?

**Comment by lolaodelola** [comment]
It seems like [2.8](https://w3ctag.github.io/societal-impact-questionnaire/#centralization) covers this issue.

**Comment by csarven** [comment]
I agree that section 2.8 covers this issue. We can however elaborate, including examples.

Perhaps we can start this section with a brief intro along the lines of:

Centralisation can occur when a feature gives disproportionate influence to a few entities, whether through control over information flows, trust decisions, or other critical functions. This can reduce openness, limit user choice, and create risks of exclusion or abuse.
